django/contrib/admin/views/main.py

```diff
@@ -49,7 +49,7 @@ class ChangeList:
     search_form_class = ChangeListSearchForm
 
     def __init__(self, request, model, list_display, list_display_links,
-                 list_filter, date_hierarchy, search_fields, list_select_related,
+                 list_filter, date_hierarchy, search_fields, search_help_text, list_select_related,
```

Comment:
Unfortunately, we cannot add a new positional argument in the middle, please move it to the end.